modules/local/nf-core-modified/iqtree.nf

```diff
@@ -39,29 +39,31 @@ process IQTREE {
         -nt AUTO \\
         -ntmax ${task.cpus} \\
         -mem \$memory \\
-        -m LG+F+G4 \\
-        $args \\
-        $fconst_args
-
-    # Identify the best number of threads
-    nt=\$(grep "BEST NUMBER" *.log | sed "s/.*: //g")
-
-    # Rename it and clean up
-    mv *.treefile guidetree.treefile
-    rm *fa.*
-
-    iqtree \\
-        -s $alignment \\
-        -nt \$nt \\
-        -mem \$memory \\
-        -m LG+C40+F+G4 \\
-        -ft guidetree.treefile \\
-        $args \\
-        $fconst_args
-
-    # Clean up
-    rm ./guidetree.treefile
+        -m $model \\
+        $args
 
+    # check if we're running a PMSF approximation - if so, do the following,
+    # treating the tree inferred above as a guide tree
+    if [ "$pmsf_model" != "none" ]; then
```

Comment:
We want this to be additive to the `iqtree` run above right? Not an if/else situation?

Reply:
It's admittedly a bit weird. We do want it to be an if-else.

Basically, if you only specify one model (the "tree_model" parameter), and "pmsf_model" is not specified, and equals "none" by default, then you just run iqtree once. If you've specified an actual second model "pmsf_model", then you run what's in this second step. 

I'm just using the if/else statement as a way to safeguard against needlessly inferring two trees, unless that's explicitly the intent of the user.